app/src/hooks/useGetCounter.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from 'react'
+
+export default function useGetCounter(seconds: number, isCountingUp: boolean) {
+  const [counterContent, setCounterContent] = useState(isCountingUp ? 0 : seconds)
+  const [isPaused, setIsPaused] = useState(false)
+
+  const handlePause = () => {
+    setIsPaused((isPaused) => !isPaused)
+  }
+
+  useEffect(() => {
+    const timer = setInterval(() => {
+      if (isPaused) return
+
+      setCounterContent((prevCounter) => {
+        if (isCountingUp) {
+          return prevCounter < seconds ? prevCounter + 1 : seconds
```

Comment:
Related to the next comment, the `1` here should not be hardcoded.
`1` could be the default value though.